config/linux.cmake

```diff
@@ -27,11 +27,8 @@
 #  Next modify the according entries, and then compile by
 #
 #  $ mkdir build; cd build
-#  $ cmake -C ../config.cmake ..
-#  $ cmake --build . --parallel 8
-#
-# You can increase the --parallel 8 argument to match the number of processor
```

Comment:
I can we can leave this sentence here but remove --parallel 8 from the command line.